Utilities/WaveFrontReader.h

```diff
@@ -509,6 +509,12 @@ class WaveFrontReader
                 LoadTexturePath(InFile, curMaterial->strEmissiveTexture, MAX_PATH);
                 curMaterial->bEmissive = true;
             }
+            else if (0 == wcscmp(strCommand.c_str(), L"map_RMA")
+                || 0 == wcscmp(strCommand.c_str(), L"map_occlusionRoughnessMetallic"))
```

Comment:
How do you feel about making it accept ``map_ORM`` or ``map_RMA`` and dumping the wordier ``map_occlusionRoughnessMetallic``?

Reply:
Yeah sounds acceptable, I went with `map_occlusionRoughnessMetallic` by your wording, but `map_ORM` would better conform to the Wavefront conventions.